debian/cloudstack-ui.install

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+/usr/share/cloudstack-ui/config.json
```

Comment:
Isn't this redundant as you also include the wildcard one line below?

In addition, config files should never be in /usr/share.

I would suggest:

- Create /etc/cloudstack/ui/config.json
- Symlink /usr/share/cloudstack-ui/config.json to /etc/cloudstack/ui/config.json

This way, when upgrading the DEB package will prompt before overriding any changes.

I suggest doing the same for the RPM packages.

Reply:
I was thinking about conffiles but this is good idea - fixed.